src/services/ServerIO/index.js

```diff
@@ -119,24 +119,26 @@ class ServerIO {
     }
 
     createRoom(id, options) {
-        const { participants, isPrivate } = Object(options);
+        const { name, participants, isPrivate } = Object(options);
```

Comment:
This is something that need to be revised, because I added to fix a specific bug on the test project chat-app